src/mailer/templates.ts

```diff
@@ -0,0 +1,83 @@
+export function signinMessage(name: string, token: string, code: string): object {
+  switch (code) {
+    case 'pl':
+      return {
+        subject: `Cześć ${name}, jedź z VIKING GARAGE`,
+        text:
+`Cześć ${name}!
+
+Witaj w VIKING GARAGE!
```

Comment:
firstly I added module with translation, but for now we have only to emails so it's more convenient to translate them like this